docs/advanced-features/react-18/switchable-runtime.md

```diff
@@ -38,22 +25,21 @@ export const config = {
-| NPM Packages                                                                                                                                        | All           | All               | A smaller subset |
-
-Next.js' default runtime configuration is good for most use cases, but there’re still many reasons to change to one runtime over the other one.
+|                                                                                                                                                     | Node (Server) | Node (Serverless) | Edge                                                     |
+| --------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- | ----------------- | -------------------------------------------------------- |
+| Name                                                                                                                                                | `nodejs`      | `nodejs`          | `edge` or `experimental-edge` if using Next.js Rendering |
+| [Cold Boot](https://vercel.com/docs/concepts/get-started/compute#cold-and-hot-boots?utm_source=next-site&utm_medium=docs&utm_campaign=next-website) | /             | ~250ms            | Instant                                                  |
+| HTTP Streaming                                                                                                                                      | Yes           | Yes               | Yes                                                      |
+| IO                                                                                                                                                  | All           | All               | `fetch`                                                  |
+| Scalability                                                                                                                                         | /             | High              | Highest                                                  |
+| Security                                                                                                                                            | Normal        | High              | High                                                     |
+| Latency                                                                                                                                             | Normal        | Low               | Lowest                                                   |
+| Code Size                                                                                                                                           | /             | 50MB              | 1MB                                                      |
```

Comment:
Code size for Edge Functions is now:

- hobby: 1MB
- oss/pro: 2MB
- enterprise: 4MB